src/main/java/org/commcare/util/EncryptionUtils.java

```diff
@@ -0,0 +1,114 @@
+package org.commcare.util;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+public class EncryptionUtils {
+
+    /**
+     * Encrypts a message using the AES encryption
+     *
+     * @param message a message to be encrypted
+     * @param key     the key used for encryption
+     * @return AES encrypted message using the given key
```

Comment:
A base64 encoded payload containing the IV and AES encrypted ciphertext, which can be decoded by this utility's decrypt method and the same symmetric key